src/common/Timer.cc

```diff
@@ -33,11 +33,7 @@ class SafeTimerThread : public Thread {
 };
 
 
-
```

Comment:
At some point it might be better just to use replace all our uses of Timer.cc with ceph_timer.h, since it already uses the new concurrency, time, functions, etc,

Reply:
yeah, we could make that change in another PR. i think we could even deprecate utime in future, and just use it as a ondisk/wire format.

Reply:
note that conversion from SafeTimer to ceph::timer isn't trivial in cases that rely on the "Safe" part